pkg/hooks/manager_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"net"
 	"os"
 	"path/filepath"
+	"strconv"
```

Comment:
Will it make sense to split the changes to template.go from the changes to virt-launcher?

Reply:
imho no need, as it is small
but i can take into account for next times